config/routes.rb

```diff
@@ -0,0 +1,15 @@
+Rails.application.routes.draw do
+  #ORDER MATTERS!
+  #new task
+  # get '/tasks/new', to: 'tasks#new', as: 'new_task'
+  resources :tasks, only: [:index, :show, :create, :new, :edit, :update, :destroy]
```

Comment:
I like that you're doing `resources`, but since you are using all 7 restful routes...

```suggestion
  resources :tasks
```